imgui.h

```diff
@@ -490,6 +490,8 @@ namespace ImGui
     // - Most widgets return true when the value has been changed or when pressed/selected
     // - You may also use one of the many IsItemXXX functions (e.g. IsItemActive, IsItemHovered, etc.) to query widget state.
     IMGUI_API bool          Button(const char* label, const ImVec2& size = ImVec2(0, 0));   // button
+    IMGUI_API bool          BeginButtonDropDown(const char* label, ImVec2 buttonSize);
```

Comment:
`const ImVec2 size  = ImVec2(0,0)`

Rather keep the API consistent with ImGui.